assets/variant-list.css

```
@@ -367,4 +422,105 @@ variant-list-remove-button .icon-remove {
   .variant-list__table thead th:first-child {
     width: 37%;
   }
+
+  .variant-list-buttons {
+    display: flex;
+    align-self: flex-start;
+    width: 37%;
+  }
+
+  variant-list-remove-all-button {
+    margin-left: 0.9rem;
+  }
+}
+
+.variant-list__total-items {
+  display: flex;
+}
+
+.variant-list__total-items h3,
+.totals__subtotal-value {
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+.variant-list__total-items p,
+.totals__subtotal {
+  margin-top: 0.2rem;
+  opacity: 0.75;
+  margin-bottom: 1.2rem;
+}
+
+.variant-list__total-items h3,
+.totals__subtotal-value {
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+.variant-list__total-items p,
+.totals__subtotal {
+  margin-top: 0.2rem;
+  opacity: 0.75;
+  margin-bottom: 1.2rem;
+}
+
+.variant-list-total__info {
+  display: flex;
+}
+
+.ariant-list-total__info,
+.variant-list-total__confirmation {
+  min-height: 10rem;
+}
+
+.variant-list-total {
+  padding-top: 2rem;
+  border-top: 0.1rem solid rgba(var(--color-foreground), .08);
+}
+
+
+@media screen and (min-width: 750px) {
+  .variant-list__total-items {
+    padding-left: calc(((11rem / var(--font-body-scale) + var(--inputs-border-width) * 2)/ 2) - 0.7rem);
+  }
+
+  .variant-list__total-items {
+    flex-direction: column;
+    align-items: center;
+  }
```

Same selectors, can combine?

```suggestion
  .variant-list__total-items {
    padding-left: calc(((11rem / var(--font-body-scale) + var(--inputs-border-width) * 2)/ 2) - 0.7rem);
    flex-direction: column;
    align-items: center;
  }
```